brigade/plugins/inventory/nsot.py

```diff
@@ -15,22 +15,38 @@ class NSOTInventory(Inventory):
         the name of the site the host belongs to.
 
     Environment Variables:
-        * ``NSOT_URL``: URL to nsot's API (defaults to ``http://localhost:8990/api``)
```

Comment:
environment variables are usually uppercase. What I'd do here is move everything you have here under env variables to Arguments (specify the type) and then under env variables add:

```
NSOT_URL: Corresponds to nsot_url argument
NSOT_EMAIL: Corresponds to nsot_email argument
...
```

Reply:
You are correct, I will correct this!